lib/attributor/types/class.rb

```diff
@@ -0,0 +1,32 @@
+require_relative '../exceptions'
+
+module Attributor
+  class Class
+    include Type
+
+    def self.native_type
+      return ::Class
+    end
+
+    def self.load(value, context=Attributor::DEFAULT_ROOT_CONTEXT, **options)
+      unless value.kind_of?(::String) || value.nil?
+        raise IncompatibleTypeError,  context: context, value_type: value.class, type: self
+      end
+
+      value = "::" + value if value[0..1] != '::'
+
+      value && eval(value)
```

Comment:
Ah, yes. I was completely ignoring the case of this being used in an API and was thinking only of the config usage.

Not sure I understand what you mean about `Class.of(MyClass)` for this? The type needs to accept any Ruby Class and it doesn't seem useful to fix it to a specific class. If it will always be MyClass, then there's no need for a config value, no? Just hard-code `MyClass`. Maybe I'm missing something?